sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessor.cs

```diff
@@ -98,41 +106,59 @@ public class EventProcessor<T> where T : BasePartitionProcessor, new()
         private Dictionary<string, PartitionOwnership> InstanceOwnership { get; set; }
 
         /// <summary>
-        ///   The running task responsible for performing partition load balancing between multiple <see cref="EventProcessor{T}" />
+        ///   The running task responsible for performing partition load balancing between multiple <see cref="EventProcessor" />
         ///   instances, as well as managing partition pumps and ownership.
         /// </summary>
         ///
         private Task RunningTask { get; set; }
 
         /// <summary>
-        ///   Initializes a new instance of the <see cref="EventProcessor{T}"/> class.
+        ///   The function to be called just before event processing starts for a given partition.
```

Comment:
nit:  How do you feel about standardizing on "handler" for these, since they're intended to kind-of-sort-of mimic the event handler pattern.

Reply:
Seems reasonable.